src/compilersupport_p.h

```diff
@@ -210,7 +210,15 @@ static inline unsigned short encode_half(double val)
         /* underflow, make zero */
         return 0;
     }
+
+//
+// bugbug: working with tinycbor folks to address this properly. 
+// Issue https://github.com/01org/tinycbor/issues/47
+//
+#pragma warning(disable:4244)
```

Comment:
I can't accept an MSVC pragma here, not without #ifndef _MSC_VER. Is there no other way of getting this? Maybe

  return (unsigned short)(sign | ((exp + 15) << 10) | mant);